taccsite_cms/contrib/taccsite_data_list/models.py

```diff
@@ -0,0 +1,99 @@
+from cms.models.pluginmodel import CMSPlugin
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from djangocms_attributes_field import fields
+
+from taccsite_cms.contrib.helpers import get_choices
+
+from .constants import ORIENTATION_DICT, TYPE_STYLE_DICT, DENSITY_DICT
+
+
+
+# Constants
+
+ORIENTATION_CHOICES = get_choices(ORIENTATION_DICT)
+TYPE_STYLE_CHOICES = get_choices(TYPE_STYLE_DICT)
+DENSITY_CHOICES = get_choices(DENSITY_DICT)
+
+
+
+# Models
+
+class TaccsiteDataList(CMSPlugin):
+    """
+    Components > "Data List" Model
+    """
+    orientation = models.CharField(
+        verbose_name=_('Orientation'),
+        help_text=_('The direction in which to lay out the data. Hint: Choose based on the amount of space available in the layout for the data.'),
+        choices=ORIENTATION_CHOICES,
+        blank=False,
+        max_length=255,
+    )
+    type_style = models.CharField(
+        verbose_name=_('Type / Style'),
+        help_text=_('The type of data to display, glossary/metadata or tabular. Notice: Each type of list has a slightly different style.'),
+        choices=TYPE_STYLE_CHOICES,
+        blank=False,
+        max_length=255,
+    )
+    density = models.CharField(
+        verbose_name=_('Density (Layout Spacing)'),
+        help_text=_('The amount of extra space in the layout. Hint: Choose based on the amount of space available in the layout for the data.'),
+        choices=DENSITY_CHOICES,
+        blank=False,
+        max_length=255,
+    )
+    truncate_values = models.BooleanField(
+        verbose_name=_('Truncate the values (as necessary)'),
+        help_text=_('Truncate values if there is not enough space to show the label and the value. Notice: Labels are truncated as necessary.'),
+        default=False,
+    )
+
+    attributes = fields.AttributesField()
+
+    def get_short_description(self):
```

Comment:
Is there a reason that you use direct indexing over `.get()` here (like in `get_classname()`).

Reply:
I have removed the feature (and use of it) in 2c174cf.

<details>
<summary>Why</summary>

- The feature was not working, even before I tired to rewrite it.
- The feature is not required for this plugin.

</details>

Reply:
Used new internal helper `get_short_description` (which uses `get`) in b30cbd0.

<details>
<summary>What about your dream of an Enum?</summary>

The dream shall remain a `TODO` comment until I am smart enough.

</details>